cmd-info.c

```diff
@@ -703,6 +706,15 @@ static int read_record_date(void *arg)
 		return -1;
 
 	info->record_date = copy_info_str(&buf[12]);
+
+	if (fgets(buf, sizeof(buf), handle->fp) == NULL)
+		return -1;
+
+	if (strncmp(buf, "elapsed_time:", 13))
+		return -1;
+
+	info->elapsed_time = copy_info_str(&buf[13]);
```

Comment:
You need to free these two data after use.

Reply:
Got it !